java/arcs/android/storage/handle/AndroidHandleManager.kt

```diff
@@ -33,7 +33,8 @@ import kotlin.coroutines.EmptyCoroutineContext
 typealias SingletonServiceStoreFactory<T> =
     ServiceStoreFactory<SingletonData<T>, SingletonOp<T>, T?>
 @UseExperimental(kotlinx.coroutines.ExperimentalCoroutinesApi::class)
-typealias SetServiceStoreFactory<T> = ServiceStoreFactory<SetData<T>, SetOp<T>, Set<T>>
+typealias SetServiceStoreFactory<T> =
```

Comment:
Why is it still using Set naming here, but changed to Collection elsewhere?

Reply:
Missed this one, thanks.